VimeoNetworking/Sources/ResponseCache.swift

```diff
@@ -113,26 +113,26 @@ final internal class ResponseCache
     
     private class ResponseMemoryCache: Cache
     {
-        private let cache = NSCache()
+        private let cache = NSCache<AnyObject, AnyObject>()
         
-        private func setResponseDictionary(responseDictionary: VimeoClient.ResponseDictionary, forKey key: String)
+        func setResponseDictionary(responseDictionary: VimeoClient.ResponseDictionary, forKey key: String)
         {
-            self.cache.setObject(responseDictionary, forKey: key)
+            self.cache.setObject(responseDictionary as AnyObject, forKey: key as AnyObject)
```

Comment:
Yeah, I'm not really sure I understand why either. I wonder if it has to do with NSCache using the `ObjectType` and `KeyType` type parameters in the method? Maybe the compiler is getting confused about the type?